docs/resources/Poll.md

```diff
@@ -0,0 +1,138 @@
+They usually are accurate, and shouldn't deviate significantly -- it's just difficult to make guarantees.
+
+To compensate for this, after a poll is finished there is a background job which performs a final, accurate tally of votes.
+This tally concludes once `is_finalized` is `true`.
+
+If `answer_counts` does not contain an entry for a particular answer, then there are no votes for that answer.
+
+###### Poll Results Object Structure
+
+| Field         | Type                                                                                                            | Description                                   |
+|---------------|-----------------------------------------------------------------------------------------------------------------|-----------------------------------------------|
+| is_finalized  | boolean                                                                                                         | Whether the votes have been precisely counted |
+| answer_counts | List of [Poll Answer Count Object](#DOCS_RESOURCES_POLL/poll-results-object-poll-answer-count-object-structure) | The counts for each answer                    |
+
+###### Poll Answer Count Object Structure
+
+| Field    | Type    | Description                                    |
+|----------|---------|------------------------------------------------|
+| id       | integer | The `answer_id`                                |
+| count    | integer | The number of votes for this answer            |
+| me_voted | boolean | Whether the current user voted for this answer |
+
+# Poll Endpoints
+
+For creating a poll, see [Create Message](#DOCS_RESOURCES_CHANNEL/create-message). After creation, the poll message cannot be edited.
+
+Apps are not allowed to vote on polls. No rights! :)
+
+## Get Answer Voters % GET /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/polls/{message.id#DOCS_RESOURCES_CHANNEL/message-object}/answers/{answer_id#DOCS_RESOURCES_POLL/poll-answer-object}
+
+Get a list of users that voted for this specific answer.
+
+###### Query String Params
+
+| Field  | Type      | Description                           | Default |
+|--------|-----------|---------------------------------------|---------|
+| after? | snowflake | Get users after this user ID          | absent  |
+| limit? | integer   | Max number of users to return (1-100) | 25      |
+
+###### Response Body
+
+| Field | Type                                              | Description                     |
+|-------|---------------------------------------------------|---------------------------------|
+| users | array of [user](#DOCS_RESOURCES_USER/user-object) | Users who voted for this answer |
+
+## Expire Poll % POST /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/polls/{message.id#DOCS_RESOURCES_CHANNEL/message-object}/expire
+
+Immediately expires (i.e. ends) the poll.
```

Comment:
expire can't be used as an active verb so this wording is weird - I'd call this End Poll (which also matches the UI text)

```suggestion
## End Poll % POST /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/polls/{message.id#DOCS_RESOURCES_CHANNEL/message-object}/expire

Immediately ends the poll, instead of waiting for the configured expiry time.
```